apps/drupal/particle.theme

```diff
@@ -87,3 +89,39 @@ function particle_preprocess_page(&$variables) {
   }
 
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function particle_theme_suggestions_form_alter(array &$suggestions, array $variables) {
+  $id = $variables['element']['#id'];
+  if ($id == 'views-exposed-form-search-page' || $id == 'search-block-form') {
+    $suggestions[] = 'form__particle_search';
+  }
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function particle_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Switch method to allow additional cases for these forms.
+  // Let's be explicit because it's easier for newcomers.
+  $id = $form['#id'];
+  switch($id) {
+    case 'views-exposed-form-search-page':
+      // Clean the unwanted label.
+      unset($form['#info']['filter-search_api_fulltext']['label']);
+      // Update type and add Bootstrap classes.
+      $form['keywords']['#type'] = 'search';
```

Comment:
Make comment that the form field name is `keywords` (this is very confusing before figuring this out)